backend/apps/recommendations/migrations/0002_alter_recommendation_content_type_ossissue.py

```
@@ -0,0 +1,37 @@
+# Generated by Django 5.0.14 on 2026-07-14 17:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('recommendations', '0001_initial'),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)